python/GafferUI/EditScopeUI.py

```diff
@@ -394,6 +388,26 @@ def __menuDefinition( self ) :
 			"/None", { "command" : functools.partial( self.getPlug().setInput, None ) },
 		)
 
+		if currentEditScope is not None :
```

Comment:
Yeah, no good answers to this at the moment. Seems we're going to end up with clutter somewhere unless we remove the functionality entirely. At least for now we only see the clutter in the global Edit Target menu and any editor-specific menus that have been overridden rather than clutter in or next to every menu...